ci/build.sh

```diff
@@ -0,0 +1,373 @@
+#!/bin/bash
```

Comment:
Is this the same file previously in ci/rpm7/build.sh? In that case, I'd suggest to use the old place (to minimize changes in the PR that are more costly to review)

Reply:
Not the same, rpm7 removed, because scenarios adapted to work with any platform and version